skt/executable.py

```diff
@@ -467,10 +383,6 @@ def cmd_run(cfg):
 
     save_state(cfg, {'retcode': retcode})
 
-    if retcode != SKT_ERROR:
-        report_results(run_result_path, 'true' if retcode else 'false',
```

Comment:
The `report_string` is still being created in the runner. Is that planned to be a separate change?

Reply:
@veruu Nope, just missed that.